samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/SwitchSample.java

```diff
@@ -361,4 +361,169 @@ public void doNotReplaceWithSwitchOnPrimitiveWrapper(Integer i1) {
             i = 30;
         }
     }
+
+    public void mergeCases(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 0:
+        case 1:
+            i = 0;
+            break;
+        default:
+            i = 1;
+            break;
+        }
+    }
+
+    public void mergeCasesWithUnbrokenCase(int i1) {
+        int i = 0;
+        switch (i1) {
+        case 1:
+            i = 1;
+        case 0:
```

Comment:
I think more logical to merge it :( :)

I think I understand why you don't want to merge it. You consider the default as an "unhandlable" situation whereas the cases are standard code. If both are identical, it is only a coincidence and it should not be merged because it has not the same meaning.

Even if we merge the code, we do not merge `case X` and `default` so we still see that there are distinct processes.